gammapy/irf/energy_dispersion.py

```diff
@@ -335,16 +325,17 @@ def write(self, filename, **kwargs):
     def get_resolution(self, e_true):
         """Get energy resolution for a given true energy.
 
-        The resolution is given as the standard deviation
-        of the logarithm of the reconstructed energy.
+        The resolution is given as a percentage of the true energy
```

Comment:
Is it possible to give a formula or better description here how energy resolution is defined / computed?
Is it simply computed as the standard deviation of the `E_reco / E_true` distribution?